core/src/test/resources/packaging/scripts/50_plugins.bats

```diff
@@ -78,7 +78,7 @@ install_package() {
     [ -e "$MARVEL_ZIP" ]
 
     # Install Marvel
-    run /usr/share/elasticsearch/bin/plugin -i elasticsearch/marvel/latest -u "file://$MARVEL_ZIP"
+    run /usr/share/elasticsearch/bin/plugin install elasticsearch/marvel/latest -u "file://$MARVEL_ZIP"
```

Comment:
Same here, can you please remove the 2 tests in this file that install Marvel?

Thanks.